### ISSUE TITLE
"Given type (PNG) is not valid" after updated to Grav 1.7.0-rc.10

### ISSUE THREAD

**Issue by dinoue** [getgrav/grav-plugin-lightslider#40]
Hello, 

After updated to Grav 1.7.0-rc.10 from rc.8,
I got a error page below,
```
Server Error
Sorry, something went terribly wrong!

0 - An exception has been thrown during the rendering of a template ("Given type (PNG) is not valid").
```

The page was written in modular format, 
`{% include 'modular/lightslider.html.twig' with {'page': page.find('/information/godo/fireservice/_slide')} %}
`
And I put PNGs into /information/godo/fireservice/_slide Folder.

The error log line is below.
I'm sorry just only a report.

`[2020-05-07 10:32:22] grav.CRITICAL: An exception has been thrown during the rendering of a template ("Given type (PNG) is not valid"). - Trace: #0 C:\inetpub\grav\vendor\twig\twig\src\Template.php(422): Twig\Template->displayWithErrorHandling(Array, Array) #1 C:\inetpub\grav\cache\twig\fb\fb84dda545aecd082015d3823be7df6ffbfac6ce2f52a688dadcafd81914e09c.php(36): Twig\Template->display(Array) #2 C:\inetpub\grav\vendor\twig\twig\src\Template.php(455): __TwigTemplate_f1fc81be788204fa9cc3c10c9fb220f5420b1005e6b0759acb204c4670934b68->doDisplay(Array, Array) #3 C:\inetpub\grav\vendor\twig\twig\src\Template.php(422): Twig\Template->displayWithErrorHandling(Array, Array) #4 C:\inetpub\grav\vendor\twig\twig\src\Template.php(434): Twig\Template->display(Array) #5 C:\inetpub\grav\vendor\twig\twig\src\TemplateWrapper.php(47): Twig\Template->render(Array, Array) #6 C:\inetpub\grav\vendor\twig\twig\src\Environment.php(384): Twig\TemplateWrapper->render(Array) #7 C:\inetpub\grav\system\src\Grav\Common\Twig\Twig.php(299): Twig\Environment->render('@Page:D:/inetpu...', Array) #8 C:\inetpub\grav\system\src\Grav\Common\Page\Page.php(903): Grav\Common\Twig\Twig->processPage(Object(Grav\Common\Page\Page), '<h2>\xE6\xB6\x88\xE9\x98\xB2\xE8\xA8\xAD\xE5\x82...') #9 C:\inetpub\grav\system\src\Grav\Common\Page\Page.php(754): Grav\Common\Page\Page->processTwig() #10 C:\inetpub\grav\system\src\Grav\Common\Twig\Twig.php(376): Grav\Common\Page\Page->content() #11 C:\inetpub\grav\system\src\Grav\Common\Service\OutputServiceProvider.php(28): Grav\Common\Twig\Twig->processSite('html') #12 C:\inetpub\grav\vendor\pimple\pimple\src\Pimple\Container.php(118): Grav\Common\Service\OutputServiceProvider->Grav\Common\Service\{closure}(Object(Grav\Common\Grav)) #13 C:\inetpub\grav\system\src\Grav\Common\Processors\RenderProcessor.php(30): Pimple\Container->offsetGet('output') #14 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\RenderProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #15 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #16 C:\inetpub\grav\system\src\Grav\Common\Processors\DebuggerAssetsProcessor.php(29): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #17 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\DebuggerAssetsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #18 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #19 C:\inetpub\grav\system\src\Grav\Common\Processors\PagesProcessor.php(71): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #20 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\PagesProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #21 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #22 C:\inetpub\grav\system\src\Grav\Common\Processors\TwigProcessor.php(29): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #23 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\TwigProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #24 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #25 C:\inetpub\grav\system\src\Grav\Common\Processors\AssetsProcessor.php(30): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #26 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\AssetsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #27 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #28 C:\inetpub\grav\system\src\Grav\Common\Processors\SchedulerProcessor.php(31): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #29 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\SchedulerProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #30 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #31 C:\inetpub\grav\system\src\Grav\Common\Processors\BackupsProcessor.php(30): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #32 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\BackupsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #33 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #34 C:\inetpub\grav\system\src\Grav\Common\Processors\TasksProcessor.php(60): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #35 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\TasksProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #36 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #37 C:\inetpub\grav\system\src\Grav\Common\Processors\RequestProcessor.php(54): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #38 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\RequestProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #39 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #40 C:\inetpub\grav\system\src\Grav\Common\Processors\ThemesProcessor.php(29): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #41 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\ThemesProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #42 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #43 C:\inetpub\grav\system\src\Grav\Common\Processors\PluginsProcessor.php(30): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #44 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\PluginsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #45 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #46 C:\inetpub\grav\system\src\Grav\Common\Processors\InitializeProcessor.php(110): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #47 C:\inetpub\grav\system\src\Grav\Common\Debugger.php(529): Grav\Common\Processors\InitializeProcessor::Grav\Common\Processors\{closure}() #48 C:\inetpub\grav\system\src\Grav\Common\Processors\InitializeProcessor.php(111): Grav\Common\Debugger->profile(Object(Closure)) #49 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(45): Grav\Common\Processors\InitializeProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #50 C:\inetpub\grav\system\src\Grav\Framework\RequestHandler\Traits\RequestHandlerTrait.php(57): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #51 C:\inetpub\grav\system\src\Grav\Common\Grav.php(249): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #52 C:\inetpub\grav\index.php(58): Grav\Common\Grav->process() #53 {main} [] []`

**Comment by rhukster** [comment]
I think I already fixed this today in Grav 1.7 branch.  Basically a .PNG image extension causing the issue.

**Comment by dinoue** [comment]
@rhukster ,
You mean...
https://github.com/getgrav/grav/commit/69078aaf497488911d127897ff02a6e0bcd20158

YES!! It worked!!
Thank you.
